### ISSUE TITLE
Full code?

### ISSUE THREAD

**Issue by zelenooki87** [caojiezhang/DATSR#2]
Hi. Thank you for great and promising project.
Unfortunately, had not success in Colab:
datsr module not found error.
Tried installing through pypi but seems to doesnt exist. Also requirements.txt missing from project.
And one more thing, if I understood, we can use Low Res image as reference? so from 1 image to get HiRes? not only getting cropped area zoomed in from reference...
expecting devs to release final code...
thank you very much

**Comment by lucellent** [comment]
They haven't posted the full release yet, you can't run it.

I'm also patiently waiting for it, looks so exciting

**Comment by codinker** [comment]
expecting final code...

**Comment by djkzxxf** [comment]
Brother, did you use the pre-training model to test the indicators to achieve the results of his paper. I measured that it did not meet the index of his paper, which was a lot worse.

**Comment by codinker** [comment]
Yes, I tried to reproduce with the previous pre-trained model. It also did not achieve the effect in the paper, which should have used multiple pictures, but the corresponding code was not published.










At 2023-03-16 16:35:21, "djkzxxf" ***@***.***> wrote:

Brother, did you use the pre-training model to test the indicators to achieve the results of his paper. I measured that it did not meet the index of his paper, which was a lot worse.
